include/alpaka/meta/NdLoop.hpp

```diff
@@ -142,7 +143,7 @@ namespace alpaka
         //! Loops over an n-dimensional iteration index variable calling f(idx, args...) for each iteration.
         //! The loops are nested in the order given by the IndexSequence with the first element being the outermost and the last index the innermost loop.
         //!
-        //! \param indexSequence A sequence of indices being a permutation of the values [0, dim-1], where every values occurs at most once.
+        //! \param indexSequence A sequence of indices being a permutation of the values [0, dim-1], where each value occurs at most once.
```

Comment:
Isn't the whole `, where ...` part redundant? A permutation of a tuple of unique elements will ofc also have unique elements.

Reply:
removed it